eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -289,10 +289,10 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
             firstPerCommitmentPoint = keyManager.commitmentPoint(localParams.channelKeyPath, 0))
           val remoteParams = RemoteParams(
             nodeId = remoteNodeId,
-            dustLimitSatoshis = open.dustLimitSatoshis,
+            dustLimitSatoshis = open.dustLimitSatoshis.toLong,
```

Comment:
Why are the amount not typed in `RemoteParams`? I think we should push those types everywhere we have an amount, don't we?

Reply:
Sure, i'm starting from the `wire` messages (LN messages) and the PR is in draft, as the changes reach more part of the code it will be upgraded to typed amounts.